docs/source/api_reference/transforms.rst

```diff
@@ -45,6 +45,7 @@ Decomposition transforms and their utilities:
    decomposition.SklearnPreprocessingPerIntervalModel
    decomposition.SklearnRegressionPerIntervalModel
    decomposition.FourierDecomposeTransform
+   decomposition.ModelDecomposeTransform
```

Comment:
We could replace it with `ModelDecomposeTransform`, because it could be imported from `etna.transforms`. We set `decomposition` prefix here only for entities that can't be imoported like that.
This notion also applies to `decomposition.FourierDecomposeTransform`.